interface/wx/appprogress.h

```diff
@@ -45,10 +45,10 @@ class WXDLLIMPEXP_CORE wxAppProgressIndicator
         Check if the application progress display is available.
 
         Currently this only returns @true when using wxMSW and running
-        under Windows 7 or later systems (which provide task bar button
+        under Windows 7 or later systems (which provides the task bar button
```

Comment:
We could just remove the mention of the version completely, couldn't we? We don't support anything earlier anyhow.

Reply:
Oh, that's right. I just updated it.